tutorials/nlp/02_NLP_Tokenizers.ipynb

```diff
@@ -0,0 +1,2276 @@
+{
```

Comment:
I think is is better not to use tokenizer.tokernizer in our code. Instead of that, it is better to user @property and @setter for such common properties as we encapsulate the tokenizer. We can get out those values from the tokenizer.tokenizer and pass it out. In the current design the internal tokenizers should all have the same properties which reduced our flexibility to support many tokenizers.

Reply:
This is just to showcase the wrapper for AutoTokenizer, so that when users want to use HuggingFace tokenizers they know how to access that. This notebook explores all possibilities with nemo tokenizers.

Reply:
But in this tutorial we are telling our users to use tokenzier.tokenzier.special_tokens to access the special tokens? How can user access the special_tokens without using tokenizer.tokenizer?

Reply:
If we do not want to add such feature in this PR, it is fine but I think we should add it to our TODO. We should not let the user to use tokenizer.tokenizer for common needs.